genie-agent/src/main/java/com/netflix/genie/agent/cli/UserConsole.java

```diff
@@ -44,6 +53,13 @@
     private static final String PID_SYSTEM_PROPERTY_NAME = "PID";
     private static final Logger LOGGER = LoggerFactory.getLogger(CONSOLE_LOGGER_NAME);
 
+    /**
+     * Because the banner is printed in the log file and not visible to the user, manually re-print it
+     * in {@code UserConsole}. Use the existing Spring configuration to control this behavior.
+     */
+    private static final String BANNER_LOCATION_SPRING_PROPERTY_KEY = "spring.banner.location";
```

Comment:
Did you find that spring didn't already have constants for these values? There might also be a `@ConfigurationProperties` you can inject for `spring.banner` that would alleviate the need for you to do this manually

Looks like `SpringApplication` has some constants:
```	/**
	 * Default banner location.
	 */
	public static final String BANNER_LOCATION_PROPERTY_VALUE = SpringApplicationBannerPrinter.DEFAULT_BANNER_LOCATION;

	/**
	 * Banner location property key.
	 */
	public static final String BANNER_LOCATION_PROPERTY = SpringApplicationBannerPrinter.BANNER_LOCATION_PROPERTY;```

Reply:
Ah! i found the constants in `SpringApplicationBannerPrinter`, but they're package private.
Didn't notice `SpringApplication` kindly making them public :-)